python/ray/serve/_private/application_state.py

```diff
@@ -51,6 +51,7 @@ def __init__(
         self.deploy_obj_ref = deploy_obj_ref
         self.app_msg = ""
         self.route_prefix = None
+        self.fastapi_docs_path = None
```

Comment:
This should not be tied directly to fastapi, we may support other docs links in the future (e.g., if there is a different HTTP wrapper or users provide their own).

Let's call it `docs_path` everywhere except in the fastapi-specific code.